src/System.CommandLine.Tests/Help/HelpBuilderTests.cs

```diff
@@ -469,6 +469,38 @@ public void Arguments_section_includes_configured_argument_aliases()
             help.Should().Contain("Sets the verbosity.");
         }
 
+
+        private enum VerbosityOptions
+        {
+            quiet,
+            q,
+            minimal,
+            m,
+            normal,
+            n,
+            detailed,
+            d,
+            diagnostic,
+            diag
+        }
+
+        [Fact]
+        public void Arguments_section_uses_name_if_suggestions_are_long()
+        {
+            var command = new Command("the-command")
+            {
+                new Option<VerbosityOptions>(new[] { "-v", "--verbosity" })
+                {
+                    ArgumentHelpName = "LEVEL"
```

Comment:
I wonder if just setting the `ArgumentHelpName` should disable the use of suggestions in help output.

Reply:
That makes sense to me.

Reply:
@jonsequitur should I switch to implementing your suggestion instead?

Reply:
That would be great.